src/test/java/teammates/test/cases/ui/StudentCourseDetailsPageActionTest.java

```diff
@@ -112,8 +117,66 @@ public void testExecuteAndPostProcess() {
         
     }
 
+    @Test
+    public void testTeamMemberDetailsOnViewTeamPage() throws EntityDoesNotExistException {
+        AccountAttributes student = dataBundle.accounts.get("student1InCourse1");
+        
+        String[] submissionParams = createValidParamsForProfile();
+        StudentProfileAttributes expectedProfile = getProfileAttributesFrom(submissionParams);
+        gaeSimulation.loginAsStudent(student.googleId);
+        
+        // adding profile picture for student1InCourse1
+        StudentProfileEditSaveAction action = getStudentProfileEditSaveAction(submissionParams);
+        RedirectResult result = (RedirectResult) action.executeAndPostProcess();
+        expectedProfile.googleId = student.googleId;
+        assertFalse(result.isError);
+        
+        StudentAttributes student1 = dataBundle.students.get("student1InCourse1");
+        
+        gaeSimulation.logoutUser();
+        gaeSimulation.loginAsStudent(dataBundle.accounts.get("student2InCourse1").googleId);
+        String[] submissionParam = new String[] {
+                Const.ParamsNames.COURSE_ID, student1.course
+        };
+        
+        StudentCourseDetailsPageAction pageAction = getAction(submissionParam);
+        ShowPageResult pageResult = getShowPageResult(pageAction);
+        StudentCourseDetailsPageData pageData = (StudentCourseDetailsPageData) pageResult.data;
+        
+        List<StudentAttributes> actualStudentsList = pageData.getStudentCourseDetailsPanel().getTeammates();
+        boolean isStudentDisplayedOnViewTeam = false;
+        for (StudentAttributes stud : actualStudentsList) {
+            if (student1.email.equals(stud.email) && student1.name.equals(stud.name)
+                    && student1.getPublicProfilePictureUrl().equals(stud.getPublicProfilePictureUrl())) {
```

Comment:
If it is done this way, we cannot be sure that there actually exists a non-default profile picture URL. The URL should be hardcoded here.